### PR TITLE
Jetpack Manage: Replace all references to 'Pro Dashboard' with 'Jetpa…

### PR DESCRIPTION
**Pull request by mattgawarecki**
This pull request updates all text copy references to "Jetpack Pro Dashboard" by replacing them with "Jetpack Manage."

Resolves https://github.com/Automattic/jetpack-genesis/issues/22.

## Proposed Changes

* Replace all string references to `(your |the )?Jetpack Pro Dashboard` with "Jetpack Manage."

### Checkout / redirect modal

* **In Calypso Blue**, visit `/checkout/jetpack/jetpack_ai_yearly?source=jetpack-plans`.
* Verify you see a modal when the page loads, and that modal references Jetpack Manage instead of Jetpack Pro Dashboard.

### Dashboard

* **In Calypso Green**, visit `/dashboard`.
  * Modify line of `client/jetpack-cloud/sections/agency-dashboard/sites-overview/index.tsx` to force `showEmptyState` to equal true.
  * Verify you see a page state that reflects no active sites (see below screenshots).
  * Verify all verbiage references Jetpack Manage instead of Jetpack Pro Dashboard.

### Licenses

* **In Calypso Green**, visit `/partner-portal/licenses`.
  * Modify line 65 of `client/jetpack-cloud/sections/partner-portal/primary/licenses/index.tsx` to force `showEmptyStateContent` to equal true; or, in Redux DevTools, dispatch the following event:

```js
{
  type: 'JETPACK_PARTNER_PORTAL_LICENSE_COUNTS_RECEIVE',
  counts: { all: 0 }
}
```
  * Verify you see a page state that reflects no active sites (see below screenshots).
  * Verify all verbiage references Jetpack Manage instead of Jetpack Pro Dashboard.

### Prices

* **In Calypso Green**, visit `/partner-portal/prices`.
* Verify all verbiage references Jetpack Manage instead of Jetpack Pro Dashboard.

### Checkout / redirect modal

Screen Shot 2023-09-11 at 14 04 48

### Dashboard

Screen Shot 2023-09-11 at 13 58 14

### Partner Portal > Licenses
Screen Shot 2023-09-11 at 13 54 19

### Partner Portal > Prices

Screen Shot 2023-09-11 at 13 54 42

### CODE REVIEW

**Review by oikeme — APPROVED** [review]
LGTM :shipit:

**Comment by yashwin** [comment]
@mattgawarecki: I think we should wait for the translations to be over before we merge this.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8931108

Thank you @mattgawarecki for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8931108).

**Pull request merged**
